buildspec_integ.yml

```diff
@@ -0,0 +1,32 @@
+version: 0.2         
+phases:
+  install:
+    runtime-versions:
+      golang: 1.12
+  pre_build:
+    commands:
+      - echo Running the integration test
+  build:
+    commands:
+      # Get the default credentials and set as environment variables
+      - 'CREDS=`curl 169.254.170.2$AWS_CONTAINER_CREDENTIALS_RELATIVE_URI`'
+      - 'export AWS_ACCESS_KEY_ID=`echo $CREDS | jq -r .AccessKeyId`'
+      - 'export AWS_SECRET_ACCESS_KEY=`echo $CREDS | jq -r .SecretAccessKey`'
+      - 'export AWS_SESSION_TOKEN=`echo $CREDS | jq -r .Token`'
```

Comment:
I think I might've been the one who originally suggested this... but I now realize that it shouldn't be necessary. The SDK/AWS CLI will query `169.254.170.2$AWS_CONTAINER_CREDENTIALS_RELATIVE_URI` for you. You should not need to set these env vars.

Reply:
I think its necessary for our integ test. Integ test is running in a separate container which I believe somhow search for credentials only in ENV variables. If we do not set these variables, our integ test for cloudwatch fails. See the following logs. 

Successfully tagged test_cloudwatch_logger-log-key:latest 
The AWS_ACCESS_KEY_ID variable is not set. Defaulting to a blank string. 
The AWS_SECRET_ACCESS_KEY variable is not set. Defaulting to a blank string. 
The AWS_SESSION_TOKEN variable is not set. Defaulting to a blank string. 
Creating network "test_cloudwatch_default" with the default driver

Reply:
Oh yeah... sorry I forgot about that.